EIPS/eip-170.md

```diff
@@ -12,12 +12,13 @@ created: 2016-11-04
 [Spurious Dragon](./eip-607.md)
 
 ### Parameters
+- `MAX_CODE_SIZE`: `0x6000` (`2**14 + 2**13`)
 - `FORK_BLKNUM`: 2,675,000
 - `CHAIN_ID`: 1 (main net)
```

Comment:
Could also do this based on https://eips.ethereum.org/EIPS/eip-2228:
```suggestion
- `CHAIN_ID`: 1 (Mainnet)
```

Reply:
Fine, but these other params are irrelevant (we don't define hard fork blocks in the EIPs nowaways).